website/common/locales/en/challenge.json

```diff
@@ -30,7 +30,7 @@
   "notParticipating": "Not Participating",
   "either": "Either",
   "createChallenge": "Create Challenge",
-  "createChallengeAddTasks": "Add Challenge Tasks",
+  "createChallengeCloneTasks": "Clone Challenge Tasks",
```

Comment:
@EmilyOng If you do `git grep createChallengeAddTasks` in your local install, you'll see that there's other code that still uses createChallengeAddTasks. Because of that, the string should not be deleted or changed. You'll want to leave createChallengeAddTasks as it is and just add the new createChallengeCloneTasks string. :)

Reply:
Okay :)